numpy/lib/tests/test_io.py

```diff
@@ -2438,6 +2438,16 @@ def test_squeeze_scalar(self):
         assert_equal((), test.shape)
         assert_equal(expected.dtype, test.dtype)
 
+    def test_ndmin_keyword(self):
+        # lets have the same behaivour of ndmin as loadtxt
+        # as they should be the same for non-missing values
+        txt = "42"
+
+        a = np.loadtxt(StringIO(txt), ndmin=1)
+        b = np.genfromtxt(StringIO(txt), ndmin=1)
```

Comment:
```suggestion
    @pytest.mark.parametrize("ndim", [0, 1, 2])
    def test_ndmin_keyword(self, ndim: int):
        # lets have the same behaivour of ndmin as loadtxt
        # as they should be the same for non-missing values
        txt = "42"

        a = np.loadtxt(StringIO(txt), ndmin=ndim)
        b = np.genfromtxt(StringIO(txt), ndmin=ndim)
```
I'd parametrize over all valid dimensions here; this should improve the test coverage a somewhat.